src/main/java/org/carlmontrobotics/robotcode2023/RobotContainer.java

```diff
@@ -120,6 +129,8 @@ private void configureButtonBindingsManipulator() {
     new POVButton(manipulatorController, Manipulator.shelfPickupPOV).onTrue(new SetArmWristGoalPreset(GoalPos.SHELF, isCube, isFront, arm));
     new POVButton(manipulatorController, Manipulator.intakeConePOV).onTrue(new SetArmWristGoalPreset(GoalPos.INTAKE, () -> false, isFront, arm));
     new POVButton(manipulatorController, Manipulator.substationPickupPOV).onTrue(new SetArmWristGoalPreset(GoalPos.STORED, isCube, isFront, arm));
+    new Trigger(() -> {return arm.getForbFlag();}).onTrue(new InstantCommand(() -> {manipulatorController.setRumble(RumbleType.kBothRumble,1);}))
```

Comment:
I might want to make the rumble amount a constant. Don't let this prevent you from merging tho.

Reply:
The previous comment was written during competition. Now that we are no longer on a time-crunch. Please move the rumble amount to a constant.